templates/login.html

```diff
@@ -19,7 +19,7 @@
   <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
   <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
+        href="/css/styles.css">
```

Comment:
This file has "/css/styles.css", but the other html files have "static/css/styles.css". Is that intentional?